lib/services/local/externalauditstorage.go

```diff
@@ -36,8 +35,6 @@ const (
 	externalAuditStoragePrefix      = "external_audit_storage"
 	externalAuditStorageDraftName   = "draft"
 	externalAuditStorageClusterName = "cluster"
-	externalAuditStorageLockName    = "external_audit_storage_lock"
```

Comment:
I will add a ConditionalAction that this lock doesn't exist to all new AtomicWrites in the backports to v15 and v14.

This should prevent e.g. a v15.2.3+ auth with these changes from racing with a v15.2.2 auth that's till using the lock

Reply:
well I guess in v14 I'll go back to using the lock since atomicwrite doesn't exist